src/options.ts

```diff
@@ -111,9 +113,29 @@ export function _setDefaultOptions(options: Partial<Options> = {}): Options {
     ...extraTracerConfig,
   };
 
+  if (options.tracesExporter === undefined) {
+    options.tracesExporter = (process.env.OTEL_TRACES_EXPORTER ||
+      TracesExporter.OTLP) as TracesExporter;
+  }
+
   // factories
-  options.spanExporterFactory =
-    options.spanExporterFactory || defaultSpanExporterFactory;
+  if (options.spanExporterFactory === undefined) {
+    if (options.tracesExporter === TracesExporter.JAEGER) {
+      options.spanExporterFactory = jaegerSpanExporterFactory;
+      options.endpoint =
+        options.endpoint ||
+        otelEnv.OTEL_EXPORTER_JAEGER_ENDPOINT ||
+        'http://localhost:14268/v1/traces';
+    } else if (options.tracesExporter === TracesExporter.JAEGER_THRIFT_SPLUNK) {
```

Comment:
I think jaeger and jaeger-thrift-splunk should have two different exporter factories each with their own default endpoints.

Reply:
Or may be a single factory that accepts the default endpoint as an argument.

Reply:
See https://github.com/signalfx/splunk-otel-js/pull/218#discussion_r693729544

Reply:
Changed that. Thanks.